helm/fdb-operator/templates/crds/foundationdbcluster.yaml

```diff
@@ -0,0 +1,477 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Will this file updated automatically when we make changes to the CRD?

Reply:
I am also not an expert here but I think we will have to make the changes manually.

Reply:
In all the other places we use it, we generate the CRD from the code. Is there a reason we can't do that with Helm? I'd rather not introduce any way for things to get out of sync if we can avoid it.